php/generate-short-url/index.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+return function ($req, $res) {
+  $payload = json_decode($req['payload'], true);
```

Comment:
```suggestion
  $payload = json_decode($req['payload'] ?? '{}', true);
```

This converts an empty payload to empty string, otherwise payload `""` would throw exception.